src/components/scenes/ScanScene.js

```diff
@@ -26,31 +27,40 @@ type Props = {
   walletId: string,
   currencyCode: string,
   qrCodeScanned: (data: string) => void,
-  parseScannedUri: (data: string) => Promise<void>,
+  parseScannedUri: (data: string, customErrorTitle: string, customErrorDescription: string) => Promise<void>,
```

Comment:
this function is a wrapper in src/connectors/scenes/ScanConnector.js,  because in this component it always should use arguments such as  `customErrorTitle ` and `customErrorDescription` I make them required

Reply:
👍🏼